content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.lastIndexOf()' 
+Description: 'Searches a string for a specified value, and returns the position of the match.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'String'
+  - 'Method'

```

Comment:
```suggestion
  - 'Methods'

```

Reply:
if in doubt check our list of tags- https://github.com/Codecademy/docs/blob/main/documentation/tags.md